src/pages/404.js

```diff
@@ -0,0 +1,133 @@
+import { Flex, List, ListItem, Text, useTheme } from '@chakra-ui/core';
+import { graphql, StaticQuery } from 'gatsby';
+import React from 'react';
+import ErrorBoundary from '../components/ErrorBoundary';
+import Image from '../components/Image';
+import Layout from '../components/Layout';
+import Link from '../components/Link';
+
+export default function NotFound() {
+  const theme = useTheme();
+  return (
+    <Layout>
+      <ErrorBoundary>
+        <StaticQuery
+          query={NotFoundLinks}
+          render={data => (
+            <Flex
+              alignItems="center"
+              justifyContent="center"
+              marginTop={['4rem', '4rem', 0, 0]}
+              marginBottom={['4rem', '4rem', 0, 0]}
+              textAlign={['center', 'center ', 'left ', 'left ']}
+              direction={['column', 'column', 'column', 'row']}
+            >
+              <Flex direction="column" maxW="335px">
+                <Text
+                  as="h1"
+                  fontFamily={theme.fonts['heading-slab']}
+                  color={theme.colors['rbb-gray']}
+                  fontWeight="900"
+                  fontSize="64px"
+                  lineHeight="77px"
+                  textTransform="uppercase"
+                  marginBottom="0.625rem"
+                >
+                  OUR BAD,
```

Comment:
I see its in the figma, but the comma at the end feels a little off and like it shouldn't be there. Not a deal breaker at all but just thought I'd point it out.

Reply:
I agree. Will bring it up to design team